reference/release-notes/1.27.0.md

```diff
@@ -0,0 +1,78 @@
+---
+orphan: true
+---
+# 1.26.0
+
+These release notes cover new features and changes in Anbox Cloud 1.27.0.
+
+Anbox Cloud 1.27.0 is a minor release. To understand minor and patch releases, see [Release notes](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/release-notes/release-notes).
+
+Please see [Component versions](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/component-versions/) for a list of updated components.
+
+## Requirements
+
+See the [Requirements](https://documentation.ubuntu.com/anbox-cloud/en/latest/reference/requirements/) for details on general and deployment specific requirements to run Anbox Cloud.
+
+## New features & improvements
+
+### Authentication and authorization
+
+In addition to authentication through mutual TLS, we now have support for OIDC based authentication in Anbox Cloud. OIDC authentication offers the ability to store a user identity in Anbox Cloud. User identities are grouped together depending on the permissions required for the identities. It is also possible to manage identities and groups at the level of the identity provider. See [TBD](tbd) for more information.
+
+### Storage: ZFS Block Mode Enabled by Default
+
+Using EXT4-based root file systems for containers is disabled by default, and ZFS-backed block-mode volumes are now used instead.
+
+Anbox Cloud uses LXD 5.21 by default. With LXD 5.21, block mode is enabled by default for instances when using ZFS storage. Block mode improves performance for I/O-intensive workloads and is particularly beneficial for Android-based containers. To support ZFS block mode requirements, the minimum disk size for instances is now 4 GB and the default disk size is set to 6 GB. This change applies to all new instance launches.
+
+With the new defaults, make sure to monitor your ZFS storage pool for capacity and performance. Also, review and update any EXT4-specific provisioning logic that allocates disks smaller than 4 GB.
```

Comment:
Ah, I forgot to updated stories on this, sorry. This won't happen this cycle, and probably not next cycle either. We can completely remove this section, sorry.